Capstone/Profile/AccountViewController.m

```diff
@@ -119,6 +119,12 @@ -(void)setUser {
     }
 }
 
+- (void)didMatchColleges:(BOOL)hide {
+    if (hide) {
+        self.takeSurveyButton.hidden = YES;
+    }
```

Comment:
since we're checking one boolean & assigning a boolean, we could have done something like this:
```suggestion
    self.takeSurveyButton.hidden = hide;
```

Reply:
resolved